dlp/src/test/java/dlp/snippets/InspectTests.java

```diff
@@ -251,6 +251,15 @@ public void testInspectStringCustomHotwordNegativeExample() throws Exception {
     assertThat(output, not(containsString("John Doe")));
   }
 
+  @Test
```

Comment:
Can you add tests for the "doctor", "Quasimodo", and "Redacted" rules used in the sample as well?

Reply:
Yes, can do.